### ISSUE TITLE
Internet Explorer 11 Display

### ISSUE THREAD

**Issue by acjcool** [facebook/fbctf#170]
![internet explorer 11](https://cloud.githubusercontent.com/assets/3783663/17056377/3f8d7810-5013-11e6-9e69-9236b7bdb35a.PNG)
Hi guys,

Any hints or tips on how to sort out the formatting/display issues on Internet Explorer 11?It seems that I can't log in as well but it's 100% with all other browsers I've tested (Chrome, Firefox, Vivaldi)

Kind regards
Juan

**Comment by rdenman** [comment]
I'm having the same issues with IE11. Can't log in, weird registration screen, drop-down menus are difficult to see, etc. When trying to log in, the 'login' button doesn't do anything, but hitting enter will take me back to the main 'Conquer the World' page. Still can't get into admin tools though.

**Comment by acjcool** [comment]
Yeah, I've tried with the Docker version as well but it doesn't seem like IE is supported for FB CTF :(
